skylib/zip.bzl

```diff
@@ -25,27 +85,7 @@ def gzip(ctx, artifact, options = None):
        the gzipped artifact.
     """
     out = ctx.actions.declare_file(artifact.basename + ".gz")
-    toolchain_info = ctx.toolchains["@io_bazel_rules_docker//toolchains/docker:toolchain_type"].info
-    input_manifests = []
-    tools = []
-    gzip_path = toolchain_info.gzip_path
-    if toolchain_info.gzip_target:
-        gzip_path = toolchain_info.gzip_target.files_to_run.executable.path
-        tools, _, input_manifests = ctx.resolve_command(tools = [toolchain_info.gzip_target])
-    elif toolchain_info.gzip_path == "":
-        fail("gzip could not be found. Make sure it is in the path or set it " +
-             "explicitly in the docker_toolchain_configure")
-
-    opt_str = " ".join([repr(o) for o in (options or [])])
-    ctx.actions.run_shell(
-        command = "%s -n %s < %s > %s" % (gzip_path, opt_str, artifact.path, out.path),
-        input_manifests = input_manifests,
-        inputs = [artifact],
-        outputs = [out],
-        use_default_shell_env = True,
-        mnemonic = "GZIP",
-        tools = tools,
-    )
+    _gzip(ctx, artifact, out, False, options, "GZIP")
```

Comment:
nit: here and below use named parameters to improve readability. Example:

```
_gzip(ctx=ctx,
         artifact=artifact,
         ...)
```

Reply:
Done.